app/tests/conftest.py

```
@@ -0,0 +1,25 @@
+
+if not hasattr(flask.helpers, "locked_cached_property"):
+    from werkzeug.utils import cached_property
+
+    flask.helpers.locked_cached_property = cached_property
+
+import app.context_processors  # noqa: F401 - patch が app.context_processors を参照する前にモジュールを読み込む
+
+from unittest.mock import patch
+
+_patch_bg_tasks = patch(
+    "app.context_processors.initialize_background_tasks", lambda x: None
```

Ruff (0.15.1)

[warning] Unused lambda argument: `x`

(ARG005)